lib/render/user.php

```diff
@@ -181,54 +181,54 @@ function RenderSiteAwards($userAwards)
             } elseif ($awardType == 2) { //    Developed a number of earned achievements
                 $tooltip = "Awarded for being a hard-working developer and producing achievements that have been earned over " . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . " times!";
 
-                $imagepath = getenv('ASSET_URL') . "/Images/_Trophy" . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . ".png";
+                $imagepath = "/Images/_Trophy" . RA\AwardThreshold::DEVELOPER_COUNT_BOUNDARIES[$awardData] . ".png";
 
                 $linkdest = ''; //    TBD: referrals page?
             } elseif ($awardType == 3) { //    Yielded an amount of points earned by players
                 $tooltip = "Awarded for producing many valuable achievements, providing over " . RA\AwardThreshold::DEVELOPER_POINT_BOUNDARIES[$awardData] . " points to the community!";
 
                 if ($awardData == 0) {
-                    $imagepath = getenv('ASSET_URL') . "/Badge/00133.png";
+                    $imagepath = "/Images/00133.png";
```

Comment:
Numbered files in the `Images` directory usually are game assets.
Do we want to rename them to something more meaningful while at it -`trophy-[color].png`?